kibbeh/package.json

```diff
@@ -17,53 +17,53 @@
     "test:watch": "jest --watch"
   },
   "dependencies": {
-    "@date-io/date-fns": "^1.3.13",
+    "@date-io/date-fns": "^2.10.8",
     "@dogehouse/kebab": "^0.2.4",
     "@material-ui/core": "^4.11.3",
     "@material-ui/pickers": "^3.3.10",
     "@svgr/webpack": "^5.5.0",
     "@tailwindcss/line-clamp": "^0.2.0",
-    "date-fns": "^2.19.0",
+    "date-fns": "^2.20.1",
     "emoji-regex": "^9.2.2",
     "formik": "^2.2.6",
     "grapheme-splitter": "^1.0.4",
     "hark": "^1.2.3",
-    "i18next": "^19.9.2",
-    "i18next-browser-languagedetector": "^6.0.1",
-    "i18next-http-backend": "^1.1.1",
+    "i18next": "^20.2.1",
```

Comment:
if you are going to upgrade major version, check to make sure there are no breaking changes

Reply:
I already checked and it appears as if translations are working

Reply:
![image](https://user-images.githubusercontent.com/55450464/114225752-633e8180-997b-11eb-85d9-6b1a94b26bc5.png)

Reply:
no I mean read the changelog for i18next

Reply:
This shouldn't break anything